src/components/PostDetails/PostDetails.scss

```diff
@@ -7,6 +7,10 @@
     text-align: right;
   }
 
+  &__visiblelist {
```

Comment:
The name of the class is visible but the work of this class is a do the element nonvisible